game/src/sandbox/dashboards/summaries.rs

```diff
@@ -385,15 +388,15 @@ fn contingency_table(ctx: &mut EventCtx, app: &App, filter: &Filter) -> Widget {
 
     Widget::col(vec![
         Text::from_multiline(vec![
-            Line("Number of slower/faster trips").small_heading(),
-            Line("by ranges of trip time (after)").small_heading(),
+            Line("Time difference by trip length").small_heading(),
+            Line("Grouped by the length of the trip before your changes."),
```

Comment:
Seems like this "(after)" was incorrect - I've updated and tried to clarify that this is a chart of accumulated time savings losses, rather than a "number of trips" count, like the previous text implied (at least to me).